pkg/kn/commands/namespaced.go

```diff
@@ -28,7 +28,7 @@ func AddNamespaceFlags(flags *pflag.FlagSet, allowAll bool) {
 		"namespace",
 		"n",
 		"",
-		"List the requested object(s) in given namespace.",
+		"If present, set the namespace scope for this CLI request",
```

Comment:
Suggestion: "Specify the namespace to operate in" (as it is self-evident that the option only has an influence 'if present'  and 'this CLI request' is so generic that it can be left out)

Reply:
+1 to suggestion above + nit: add a dot (.) at the end of the statement to align with other flags' descriptions.
I see, kubectl is using this help text
```
 kubectl options | grep namespace
  -n, --namespace='': If present, the namespace scope for this CLI request
```

Reply:
Fixed in the new version.